app/domain/errors.rb

```diff
@@ -50,7 +50,7 @@ module Authentication
     )
 
     InvalidOrigin = ::Util::TrackableErrorClass.new(
-      msg: "Invalid origin",
+      msg: "User is not authorized to login from the current origin",
```

Comment:
@shulifink please review

Reply:
This is the same message that we printed before.